addons/binding/org.openhab.binding.mihome/ESH-INF/thing/channel.xml

```diff
@@ -142,10 +142,10 @@
 		<state readOnly="true"></state>
 	</channel-type>
 
-	<channel-type id="isOpenAlarm">
+	<channel-type id="alarm">
 		<kind>trigger</kind>
-		<label>Opened window alarm</label>
-		<description>Triggers ALARM event, when window sensor is not getting "closed" for configured amount of time</description>
+		<label>Alarm trigger</label>
```

Comment:
We have the (undocumented...) best practice to keep labels always capitalised: "Alarm Trigger".